docs/changes.rst

```diff
@@ -30,6 +30,11 @@ Released: not yet
 
 **Bug fixes:**
 
+* Fixed a bug in the metric definition file that lead to
+  HTTP error 400,14 "'absolute-ifl-capping'' is not a valid value for the
```

Comment:
```suggestion
  HTTP error 400.14 "'absolute-ifl-capping' is not a valid value for the
```

Reply:
Well, the extra trailing single quote was the bug, and it appeared exactly like that in the error message.